Gruntfile.js

```diff
@@ -238,33 +46,36 @@ module.exports = function(grunt) {
           port: 9696
         }
       }
+    },
+
+    run: {
```

Comment:
Elsewhere we've been using [grunt-shell](https://www.npmjs.com/package/grunt-shell) for executing commands in grunt - looks like we can just swap this one out for that